applications/vanilla/controllers/class.discussioncontroller.php

```diff
@@ -158,6 +172,14 @@ public function index($DiscussionID = '', $DiscussionStub = '', $Page = '') {
         if (empty($canonicalUrl)) {
             $this->canonicalUrl(discussionUrl($this->Discussion, pageNumber($this->Offset, $Limit, 0, false)));
         } else {
+            $canonicalMessage = new Message(
+                sprintf(
+                    \Gdn::locale()->translate('This discussion has a more <a href="%1$s">recent version</a>.'),
```

Comment:
Do we have a PR moving adding this to the locales repo?

Reply:
We do now! See vanilla/locales#118